Sources/Private/MainThread/NodeRenderSystem/Nodes/RenderNodes/GradientStrokeNode.swift

```diff
@@ -140,7 +140,7 @@ final class GradientStrokeNode: AnimatorNode, RenderNode {
 
     /// Get dash lengths
     let dashLengths = strokeProperties.dashPattern.value.map { $0.cgFloatValue }
-    if dashLengths.count > 0 {
+    if dashLengths.count > 0 && !dashLengths.allSatisfy({ $0.isZero }) {
```

Comment:
Looking at the lint output I think it wants this to be:

```suggestion
    if dashLengths.count > 0, !dashLengths.allSatisfy({ $0.isZero }) {
```